api/API_OVERVIEW.md

```diff
@@ -76,8 +76,7 @@ See [here](CONTRIBUTING.md#api-changes) for the v2 API change process.
 Unless otherwise stated, the APIs with the same names as v1 APIs have a similar role.
 
 * [Cluster Discovery Service (CDS)](envoy/api/v2/cds.proto).
-* [Endpoint Discovery Service (EDS)](envoy/api/v2/eds.proto). This has the same role as SDS in the [v1 API](https://www.envoyproxy.io/docs/envoy/latest/api-v1/cluster_manager/sds),
-  the new name better describes what the API does in practice. Advanced global load balancing capable of utilizing N-dimensional upstream metrics is now supported.
+* [Endpoint Discovery Service (EDS)](envoy/api/v2/eds.proto). The new name better describes what the API does in practice. Advanced global load balancing capable of utilizing N-dimensional upstream metrics is now supported.
```

Comment:
This API supports dynamically fetching endpoints. See the EDS documentation

with a link to https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/service_discovery#arch-overview-service-discovery-types-eds